packages/mask/src/plugins/Trader/SNSAdaptor/trader/InputTokenPanel.tsx

```diff
@@ -1,29 +1,26 @@
 import { memo, useMemo } from 'react'
 import BigNumber from 'bignumber.js'
-import type { ChainId, SchemaType } from '@masknet/web3-shared-evm'
-import { useFungibleTokenPrice } from '@masknet/web3-hooks-base'
+import type { ChainId } from '@masknet/web3-shared-evm'
+import { useFungibleTokenPrice, useNetworkContext } from '@masknet/web3-hooks-base'
 import type { FungibleToken } from '@masknet/web3-shared-base'
-import { NetworkPluginID } from '@masknet/shared-base'
 import { InputTokenPanelUI } from './components/InputTokenPanelUI.js'
 import type { SelectTokenChipProps } from '@masknet/shared'
+import type { Web3Helper } from '@masknet/web3-helpers'
 
 export interface InputTokenPanelProps extends withClasses<'root'> {
     balance: string
     amount: string
     chainId: ChainId
     maxAmount: string
-    token?: FungibleToken<ChainId, SchemaType> | null
+    token?: FungibleToken<Web3Helper.ChainIdAll, Web3Helper.SchemaTypeAll> | null
```

Comment:
Type `FungibleToken<Web3Helper.ChainIdAll, Web3Helper.SchemaTypeAll>` is using a lot. Please add `FungibleTokenAll` and `NonFungibleTokenAll` in `@masknet/helpers`.